src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -1215,17 +1258,40 @@ function ChartValuesView({
                                 invalidAppNameMessage={commonState.invalidAppNameMessage}
                             />
                         )}
-                        {!isExternalApp &&
-                            ((!isDeployChartView && commonState.selectedProject) ||
-                                (isDeployChartView && serverMode === SERVER_MODE.FULL)) && (
-                                <ChartProjectSelector
-                                    isDeployChartView={isDeployChartView}
-                                    selectedProject={commonState.selectedProject}
-                                    handleProjectSelection={handleProjectSelection}
-                                    projects={commonState.projects}
-                                    invalidProject={commonState.invalidProject}
-                                />
-                            )}
+
+                        {!isDeployChartView && (
+                            <div className="mb-16">
+                                <div className="fs-12 fw-4 lh-20 cn-7">Project</div>
+                                <div className="flex left dc__content-space fs-13 fw-6 lh-20 cn-9">
+                                    {appMetaInfo?.projectName ? appMetaInfo?.projectName : 'unassigned'}
```

Comment:
Can use HELM_APP_UNASSIGNED_PROJECT instead of 'unassigned'

Reply:
Actually apps with no project are given unassigned in user permissions also, so this name has been derived from there.

Reply:
Then change the name to more relevant and use that const globally